website/blog/2024-08-18-introduce-tabby-inline-edit/index.mdx

```diff
@@ -0,0 +1,94 @@
+---
+authors: [wei]
+tags: [editor integration, product]
+image: ./tabby-edit.png
+
+---
+
+
+# Introduce the Inline Edit in Tabby
+
+![Tabby Edit](./tabby-edit.png)
+
+## What is Tabby Edit?  
+  
+Tabby Edit is a feature that allows you to edit your code directly in your favorite editor.  
```

Comment:
Let's always refer to the official name (Inline Edit) when talking about this feature